ce_main_app/native/scsi.cpp

```diff
@@ -665,6 +666,25 @@ void Scsi::SCSI_Inquiry(void)
         return;
     }
 
+    if(devInfo[acsiId].attachedMediaIndex != -1) {
+        int type = attachedMedia[devInfo[acsiId].attachedMediaIndex].hostSourceType;
+        switch(type) {
+        case SOURCETYPE_IMAGE:
+          memcpy(type_str, " IMG", 4);
+          break;
+        case SOURCETYPE_IMAGE_TRANSLATEDBOOT:
+          memcpy(type_str, "BOOT", 4);

```

Comment:
I would call this something like 'CEDD' or 'CE_DD' or 'CE_DD BOOT' (depending on which is the longest which will fit), because in ACSI / SCSI IDs you choose between 'sd', 'raw' and 'ce_dd' - it would be more consistent and more self explanatory.